dev/release/verify-release-candidate.sh

```diff
@@ -526,6 +530,13 @@ test_binary_distribution() {
 # To deactivate one test, deactivate the test and all of its dependents
 # To explicitly select one test, set TEST_DEFAULT=0 TEST_X=1
 : ${TEST_DEFAULT:=1}
+
+# For selective integration testing, set TEST_INTEGRATION_SELECTED=1 TEST_X=1
+: ${TEST_INTEGRATION_SELECTED:=0}
```

Comment:
This is only necessary if we want to keep the ability to have `TEST_INTEGRATION` automatically activate dependents, e.g. the following would run integration tests for C++, Java, JS, and Go.
```sh
TEST_DEFAULT=0 \
TEST_INTEGRATION=1 \
dev/release/verify-release-candidate.sh source 0.15.0 2
```
I'd be fine with changing this so `TEST_INTEGRATION` does not activate dependents and tests only for what is selected, but not sure what others think